src/streamlink/plugins/canalplus.py

```diff
@@ -6,23 +6,36 @@
 
 
 class CanalPlus(Plugin):
-    API_URL = 'http://service.canal-plus.com/video/rest/getVideos/{0}/{1}?format=json'
-    CHANNEL_MAP = {'canalplus': 'cplus', 'c8': 'd8', 'cstar': 'd17', 'cnews': 'itele'}
+	# NOTE : no live url for the moment
```

Comment:
don't use tabs, use spaces.

And the live channels use drm, i don't think there will be support for them again.

Reply:
old api still available for live channels. http://service.canal-plus.com/video/rest/getVideos/d8/785992?format=json